### PR TITLE
make the "if" sentence to fit the golang usage.

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: yupengzte <yu.peng36@zte.com.cn>

**- What I did**
Make the "if" sentence to fit the golang usage.
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by duglin** [comment]
I actually find this new form a bit harder to read.

File: daemon/graphdriver/btrfs/btrfs.go [hunk 1]
```diff
@@ -191,9 +191,8 @@ func subvolCreate(path, name string) error {
 		args.name[i] = C.char(c)
 	}
 
-	_, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
-		uintptr(unsafe.Pointer(&args)))
-	if errno != 0 {
+	if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
```
Comment (ripcurld0) [inline daemon/graphdriver/btrfs/btrfs.go]: Maybe put this in one line? or put the `errno != 0` part in the second line?

Comment (YuPengZTE) [inline daemon/graphdriver/btrfs/btrfs.go]: That is great. Thanks @ripcurld0 .
Changes had done, is it OK?
PTAL, @ripcurld0 @duglin @AkihiroSuda .

Comment (AkihiroSuda) [inline daemon/graphdriver/btrfs/btrfs.go]: Even worse; it doesn't fit on my screen 😭

Comment (ripcurld0) [inline daemon/graphdriver/btrfs/btrfs.go]: 😞 Still doesn't look good. Sorry for suggesting that.

Comment (YuPengZTE) [inline daemon/graphdriver/btrfs/btrfs.go]: Thanks, everyone.
:cry: ,it is diffcult to fit the golang usage and code readability.
It is better to close this PR, and keep the code as it is, I think.
If have any good method, I will change this code and the code in this picture: 
image

**Comment by duglin** [comment]
On May 24, 2017, at 4:17 AM, Akihiro Suda <notifications@github.com> wrote:
> 
> @AkihiroSuda commented on this pull request.
> 
> In daemon/graphdriver/btrfs/btrfs.go <https://github.com/moby/moby/pull/33348#discussion_r118189573>:
> 
> > @@ -191,9 +191,8 @@ func subvolCreate(path, name string) error {
>  		args.name[i] = C.char(c)
>  	}
>  
> -	_, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
> -		uintptr(unsafe.Pointer(&args)))
> -	if errno != 0 {
> +	if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
> Even worse; it doesn't fit on my screen 😭
> 
> 
> 
LOL I wasn’t going mention it, but yea that too :-)
Long live 80 column screens!!!

**Comment by feifeigd** [comment]
一点卵用都没有

**Comment by thaJeztah** [comment]
There's a lot of duplication though;

```go
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,     uintptr(unsafe.Pointer(&args))); errno != 0 {
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SNAP_CREATE_V2,    uintptr(unsafe.Pointer(&args))); errno != 0 {
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SNAP_DESTROY,      uintptr(unsafe.Pointer(&args))); errno != 0 {
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_QUOTA_CTL,         uintptr(unsafe.Pointer(&args))); errno != 0 {
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_QUOTA_CTL,         uintptr(unsafe.Pointer(&args))); errno != 0 {
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_QUOTA_RESCAN_WAIT, uintptr(unsafe.Pointer(&args))); errno != 0 {
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_QGROUP_LIMIT,      uintptr(unsafe.Pointer(&args))); errno != 0 {
if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_INO_LOOKUP,        uintptr(unsafe.Pointer(&args))); errno != 0 {
```

Would a utility function be an option for this?

**Comment by YuPengZTE** [comment]
Thanks @thaJeztah.
Good idea, I will try to do it.

**Pull request closed without merging**